spec/emitter_spec.rb

```diff
@@ -729,9 +728,45 @@
         "thread=c at=start\n",
         "line=3\n",
         "thread=c at=finish elapsed=0.0000\n"
-      )
+      ) # the line "thread=e line=6" is not here b/c that was in a `#silence` block
 
       expect(other_lines).to contain_exactly("thread=d line=5\n")
     end
+
+    describe "#silence!" do
+      after do |example|
+        emitter.unsilence!
+      end
+
+      it "only affects the current thread" do
+        emitter.silence!
+
+        1_000.times do
+          emitter.log(thread: :main, line: 1)
+        end
+
+        thread_b = Thread.new {
```

Comment:
Related to https://github.com/heroku/l2meter/pull/14/files#r676897496 I would name this "unsilenced_thread"